walkeeper/src/timeline.rs

```diff
@@ -109,13 +115,18 @@ impl SharedState {
             acc.hs_feedback.xmin = min(acc.hs_feedback.xmin, state.hs_feedback.xmin);
             acc.hs_feedback.catalog_xmin =
                 min(acc.hs_feedback.catalog_xmin, state.hs_feedback.catalog_xmin);
-            acc.disk_consistent_lsn = Lsn::min(acc.disk_consistent_lsn, state.disk_consistent_lsn);
-            // currently not used, but update it to be consistent
-            acc.last_received_lsn = Lsn::min(acc.last_received_lsn, state.last_received_lsn);
-            // When at least one replica has preserved data up to remote_consistent_lsn,
-            // safekeeper is free to delete it, so chose max of all replicas.
-            acc.remote_consistent_lsn =
-                Lsn::max(acc.remote_consistent_lsn, state.remote_consistent_lsn);
+
+            // Here we assume that only one active pageserver
```

Comment:
What is the definition of "active" here? How this plays with tenant relocation/possible second pageserver for failover? Why we cant continue using value calculated across all replicas?

Reply:
"Active" here means pageserver that is connected to the safekeeper and sends the feedback to it.
I'll reword the comment to clarify that.

If there is more than one active pageserver, we must choose what feedback to pass to the compute node.
So we need to identify the pageserver to which the compute node is connected, and use its feedback.

`remote_consistent_lsn` can be max of all values, though.
As for 'last_received_lsn' we need to track it for each pageserver separately, to ensure that they are caught up. 
I recall, your PR addressed that.